index.html

```diff
@@ -8,22 +8,146 @@
+            love with  my work gallery, and you are planning for one or 
+           you need a code mentor. AMBROSE's CODELAB get you covered</p>
+        <button type="button">Start colaboration</button>
+        <div id="second-section-bg-img"></div>
+  </main>
+    <!-- project section -->
+  <section>
+    <div id="project-container">
+      <h2 id="project">Projets</h2>
+    <div id="project-section1" class="project-list"></div>
+    <div class="list-item-container">
+      <div class="list-item-project" >
+       <h4>Project name goes here</h4>
+       <ul>
+         <li class=list-item1>HTML/CSS</li>
+         <li class=list-item2>Ruby on Rails</li>
+         <li class=list-item3>Javascricpt</li>
+       </ul>
+       <div class="button" >
+       <button type="button" class="img-btn">See this project <img src="./images/arrow-right.png" alt=""></button>
+      </div>
+      </div>
+    </div>
+    <div id="project-section3" class="project-list"></div>
+    <div class="list-item-container">
+      <div class="list-item-project" >
+       <h4>Project name goes here</h4>
+       <ul>
+         <li class=list-item1>HTML/CSS</li>
+         <li class=list-item2>Ruby on Rails</li>
+         <li class=list-item3>Javascricpt</li>
+       </ul>
+       
+       <button type="button" class="img-btn">See this project <img src="./images/arrow-right.png" alt=""></button>
+      </div>
+    </div>
+    <div id="project-section5" class="project-list"></div>
+    <div class="list-item-container">
+      <div class="list-item-project" >
+       <h4>Project name goes here</h4>
+       <ul>
+         <li class=list-item1>HTML/CSS</li>
+         <li class=list-item2>Ruby on Rails</li>
+         <li class=list-item3>Javascricpt</li>
+       </ul>
+       <button type="button" class="img-btn">See this project <img src="./images/arrow-right.png" alt=""></button>
+      </div>
+    </div>
+    <div  id="project-section7" class="project-list"></div>
+    <div class="list-item-container">
+      <div class="list-item-project" >
+       <h4>Project name goes here</h4>
+       <ul>
+         <li class=list-item1>HTML/CSS</li>
+         <li class=list-item2>Ruby on Rails</li>
+         <li class=list-item3>Javascricpt</li>
+       </ul>
+       <button type="button" class="img-btn">See this project <img src="./images/arrow-right.png" alt=""></button>
+      </div>
+    </div>
+    <div id="project-section9" class="project-list"></div>
+    <div class="list-item-container">
+      <div class="list-item-project" >
+       <h4>Project name goes here</h4>
+       <ul>
+         <li class=list-item1>HTML/CSS</li>
+         <li class=list-item2>Ruby on Rails</li>
+         <li class=list-item3>Javascricpt</li>
+       </ul>
+       <button type="button" class="img-btn">See this project <img src="./images/arrow-right.png" alt=""> </button>
+      </div>
+    </div>
+    <div id="project-section11" class="project-list"></div>
```

Comment:
- [ ] Your project has an issue of fitting on a mobile screen. try to make it fit in the mobile version so that someone can not scroll on the right.  Check on your margins and image width to fix the issue.  This will make your website more responsive. 
![image](https://user-images.githubusercontent.com/43877689/170816000-b1d75ef7-ed4a-47bd-8036-cbfdb60deb0b.png)

Reply:
--Done